config/app.php

```diff
@@ -120,6 +120,16 @@
 
 	'db_log_sql' => (bool) env('DB_LOG_SQL', false),
 
+	/*
+	|--------------------------------------------------------------------------
+	| Skip diagnostics checks
+	|--------------------------------------------------------------------------
+	|
+	| Allows to define class names of diagnostics checks that will be skipped.
+	|
+	*/
+	'skip_diagnostics_checks' => explode(',', env('SKIP_DIAGNOSTICS_CHECKS', '')),
```

Comment:
Please add `#SKIP_DIAGNOSTICS_CHECKS=` and short description in `.env.example` :)